requirements-dev.txt

```diff
@@ -16,7 +16,7 @@ colorama==0.4.6
 coverage==7.3.1
 craft-archives==1.1.3
 craft-cli==2.1.0
-craft-parts==1.25.1
+craft-parts@git+https://github.com/canonical/craft-parts.git@snap-bin-path
```

Comment:
do we still need this? I thought overlayfs would come from libexec

Reply:
That one if from `apt`, unless we want to move it to `libexec` by hand?

Reply:
apt for build-packages? that should come from the host

Reply:
from what I just saw in the PR, we are only using that code in craft-parts for the fuse overlayfs command, so this should be safe to drop now that that is in libexec.

Do NOT move apt to libexec.

Reply:
No, I mean the overlayfs is installed by apt in the parts, and that command is in `$SNAP/usr/bin`. We need to move it into libexec in some way.

Reply:
yes, this is why I said we want to "organize" that component into libexec

Reply:
can anyone remind me why we chose not to provision `fuse-overlayfs` in the build instance?

Reply:
update: we talked a bit about this during standup and the reason was "destructive mode"
so the agreed-upon solution is to use `organize` to put fuse-overlayfs in libexec (as Sergio said last)

Reply:
Got it.